src/patients/GeneralInformation.tsx

```diff
@@ -209,52 +241,460 @@ const GeneralInformation = (props: Props) => {
       </Panel>
       <br />
       <Panel title={t('patient.contactInformation')} color="primary" collapsible>
-        <div className="row">
-          <div className="col">
-            <TextInputWithLabelFormGroup
-              label={t('patient.phoneNumber')}
-              name="phoneNumber"
-              value={patient.phoneNumber}
-              isEditable={isEditable}
-              onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
-                onInputElementChange(event, 'phoneNumber')
-              }}
-              feedback={t(error?.phoneNumber)}
-              isInvalid={!!error?.phoneNumber}
-              type="tel"
-            />
-          </div>
-          <div className="col">
-            <TextInputWithLabelFormGroup
-              label={t('patient.email')}
-              placeholder="email@email.com"
-              name="email"
-              value={patient.email}
-              isEditable={isEditable}
-              onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
-                onInputElementChange(event, 'email')
-              }}
-              type="email"
-              feedback={t(error?.email)}
-              isInvalid={!!error?.email}
-            />
-          </div>
-        </div>
-        <div className="row">
-          <div className="col">
-            <TextFieldWithLabelFormGroup
-              label={t('patient.address')}
-              name="address"
-              value={patient.address}
-              isEditable={isEditable}
-              onChange={
-                (event: React.ChangeEvent<HTMLTextAreaElement>) =>
-                  onFieldChange && onFieldChange('address', event.currentTarget.value)
-                // eslint-disable-next-line react/jsx-curly-newline
-              }
-            />
-          </div>
-        </div>
+        <Panel title={t('patient.phoneNumber.panel')} color="primary" collapsible>
+          <table className="table table-hover">
+            <thead className="thead-light">
+              <tr>
+                <th>Type</th>
+                <th>PhoneNumber</th>
+              </tr>
+            </thead>
+            <tbody>
+              {patient.id &&
+                patient.phoneNumber &&
+                patient.phoneNumber.map((a: PhoneNumber, index: number) => (
+                  <tr key={a.id}>
+                    <td>
+                      <SelectWithLabelFormGroup
+                        name="phoneNumberType"
+                        label={t('patient.phoneNumber.type')}
+                        value={a.type}
+                        isEditable={isEditable}
+                        options={[
+                          { label: 'Home', value: 'Home' },
+                          { label: 'Work', value: 'Work' },
+                          { label: 'Temp', value: 'Temp' },
```

Comment:
The label for `Temp` should probably be `Temporary`. This is my bad, I didn't catch this when I wrote the issue.